### DISCUSSION TITLE
fnf JS engine no lag

### DISCUSSION THREAD

**Discussion by ConbiFan** [JordanSantiagoYT/FNF-JS-Engine#732] [Q&A]
How can I eliminate lag?
(Other than Rendering Mode)

**Comment by TheStinkern** [comment]
change the optimization settings
or
close some unused windows

**Reply by TheStinkern** [reply]
this includes the graphic settings

**Comment by imavie** [comment]
uh Arch Linux

**Reply by ConbiFan** [reply]
???

**Reply by imavie** [reply]
just get a new pc for christmas or use a entirely different os

**Reply by ConbiFan** [reply]
Sorry, I won't do it.

**Reply by moxie-coder** [reply]
> > just get a new pc for christmas or use a entirely different os
> 
> Bro all your answers are shit, like, sorry but half the time you are arguing or telling ppl to make obvious things.

I can agree with that, it’s usually obvious (I hate having to state the obvious honestly) or unhelpful responses from you, that being an example, and generally you should be more respectful of what you say when giving responses to an issue
